pyproject.toml

```diff
@@ -22,7 +22,7 @@ openpyxl = "2.6.4"
 rstr = "2.2.6"
 requests = "^2.18.0"
 dictionaryutils = "^3.0.2"
-py2neo = {url = "https://cdistest-public-test-bucket.s3.amazonaws.com/py2neo-py2neo-2.0.8.tar.gz"}
+psqlgraph = "^3.0.1"
```

Comment:
Could do ~3.0 if we want to catch any patch version updates, but not a big deal. Either way we'd have to regenerate the lock if we bumped the dep 😛

Reply:
That's a good idea 🤯 !